src/components/manage/Widgets/RecurrenceWidget/RecurrenceWidget.jsx

```diff
@@ -470,12 +470,12 @@ class RecurrenceWidget extends Component {
   getDefaultUntil = (freq) => {
     const moment = this.moment;
     var end = this.props.formData?.end
-      ? toISOString(this.getUTCDate(this.props.formData.end).toDate())
+      ? moment(new Date(this.props.formData?.end)).format('YYYY-MM-DD')
```

Comment:
Checking `formData?` with the `?` is not needed because it is already checked in the previous line.